applications/services/gui/modules/variable_item_list.c

```diff
@@ -220,6 +328,65 @@ void variable_item_list_set_header(VariableItemList* variable_item_list, const c
         true);
 }
 
+static bool zapper_menu_input_handler(InputEvent* event, void* context) {
```

Comment:
in this (and draw) func, if it changes real freq from cc1101 part, then probably need a real buffer, instead of doing "change 4 times and paint", to prevent flood to 1101 and cause err-ish. 

but from my read, it seems like it only save - not changing, unless back-button as call back. so should be OK and saved more RAM. but need check more (maybe)